src/test/java/org/folio/qm/controller/RecordsEditorAsyncIT.java

```diff
@@ -356,28 +356,6 @@ void testUpdateReturn422WhenRecordWithMultiple001(String filePath) throws Except
     // Now we add the new 001 field to the record and try to update existing record
     quickMarcJson.getFields().add(new FieldItem().tag("001").content("$a test value"));
 
-    putResultActions(recordsEditorResourceByIdPath(VALID_PARSED_RECORD_ID), quickMarcJson)
-      .andExpect(status().isUnprocessableEntity())
-      .andExpect(jsonPath("$.issues.size()").value(1))
-      .andExpect(jsonPath("$.issues[0].tag").value("001[1]"))
-      .andExpect(jsonPath("$.issues[0].severity").value("error"))
-      .andExpect(jsonPath("$.issues[0].definitionType").value("field"))
-      .andExpect(jsonPath("$.issues[0].message").value("Field is non-repeatable."));
-  }
-
-  @Test
-  void testUpdateReturn422WhenHoldingsRecordWithMultiple001() throws Exception {
-    log.info("===== Verify PUT record: 001 tag check =====");
-
-    mockPut(changeManagerResourceByIdPath(VALID_PARSED_RECORD_DTO_ID), SC_ACCEPTED, wireMockServer);
-
-    QuickMarcEdit quickMarcJson = readQuickMarc(QM_RECORD_EDIT_HOLDINGS_PATH, QuickMarcEdit.class)
-      .parsedRecordDtoId(VALID_PARSED_RECORD_DTO_ID)
-      .externalId(EXISTED_EXTERNAL_ID);
-
-    // Now we add the new 001 field to the record and try to update existing record
-    quickMarcJson.getFields().add(new FieldItem().tag("001").content("$a test value"));
-
     putResultActions(recordsEditorResourceByIdPath(VALID_PARSED_RECORD_ID), quickMarcJson)
```

Comment:
This cases still valid.
There should be no changes to edit, only for create operaion.

Reply:
New tests added without changing old ones.